css-properties-values-api/Overview.bs

```diff
@@ -911,6 +911,160 @@ and affects how the custom property calculates its [=computed value=].
 			return the result.
 </div>
 
+The <dfn>@property</dfn> rule {#at-property-rule}
+=================================================
+
+The ''@property'' rule provides an alternative way to register a custom
+property. Valid ''@property'' rules result in a
+[=register a custom property|registered custom property=], as if
+{{registerProperty()}} had been called with equivalent parameters.
+
+The semantics of registered properties are the same regardless of
+the mechanism used to perform the registration. This means that, once
+registered, it does not matter whether the registration originated from
+{{registerProperty()}} or ''@property'': the property has the same
+behavior either way.
+
+The syntax of ''@property'' is:
+
+	<pre class="prod def" nohighlight>
+	@property <<custom-property-name>> {
+		<<declaration-list>>
+	}
+	</pre>
+
+''@property'' rules require a 'syntax', and 'inherits' descriptor; if either are
+missing, the ''@property'' rule is invalid, and must not result in a registration.
+The 'initial-value' descriptor is optional only if the syntax is the
+[=universal syntax descriptor=], otherwise the descriptor is required; if it's
+missing, the ''@property'' rule is invalid,
+and must not result in a registration.
+
+The ''@property'' rule accepts the descriptors defined in this chapter.
+Declarations that contain other descriptors not defined here, must be ignored to
+maintain forwards compatiblity.
+
+Issue(845): What happens when we have both an ''@property'' rule, and a
+{{registerProperty()}} call for the same property?
+
+Issue(846): What happens when ''@property'' appears inside a shadow tree?
+
+The 'syntax' descriptor {#the-syntax-descriptor}
+------------------------------------------------
+
+	<pre class='descdef'>
+	Name: syntax
+	Value: <<string>>
+	For: @property
+	Initial: "*"
```

Comment:
`*` is wrong syntax, since means `zero or more occurrences` multiplier with missed term before it. Looks like it should be `Nothing` like for other descriptors if it's a required property, or `"<declaration-value>"` if it will become optional.

Reply:
It's not `*`, it's `"*"`.

Reply:
Yeah, I mean wrong syntax for a custom property value (inside quotes used specific grammar). So, `*` is not valid for that

Reply:
This accepts a `<string>`, hence `"*"` is valid in that sense.

What is or isn't a valid _syntax descriptor_ is determined by [3.4.2. Consume a syntax descriptor](https://drafts.css-houdini.org/css-properties-values-api-1/#consume-syntax-descriptor), not css3-values (directly).

In other words, a single `*` alone is a special snowflake.

Reply:
I see, thank you for clarification!